ovos_plugin_manager/templates/language.py

```diff
@@ -15,6 +15,16 @@ def detect(self, text):
     def detect_probs(self, text):
         return {self.detect(text): 1}
 
+    @property
+    def available_languages(self) -> set:
+        """Return languages supported by this detector implementation in this state
+        This property should be overridden by the derived class to advertise
+        what languages that engine supports.
```

Comment:
what does this mean here? is this the languages that can be detected by this plugin? should be a bit clearer in the docs as this is ambiguous since input/output lang is basically the same thing here

Reply:
I think any detector should have a set of languages it is able to detect. I'll update the docstring to better describe this